raiden/api/v1/encoding.py

```diff
@@ -35,16 +35,15 @@
 )
 
 
-# type converter for the flask route
 class HexAddressConverter(BaseConverter):
-
     def to_python(self, value):
-        value = address_decoder(value)
-        return value
+        if value[:2] == '0x':
```

Comment:
This makes our input to the API endpoint stricter. Before by using the `address_decoder()` we were not enforcing the `0x` prefix. We accepted hex addresses both with and without it. Do we really want to enforce this?